### PR TITLE
[JBPM-9877] Revert schema changes for DB partition

### PR DESCRIPTION
**Pull request by elguardian**
Jira: https://issues.redhat.com/browse/JBPM-9877

### CODE REVIEW

**Review by gmunozfe — CHANGES_REQUESTED** [review]
Check compilation error (end doesn't exist):
https://github.com/kiegroup/jbpm/blob/fdb0059ce00a327f0e29ba54d1484709f17229d3/jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/impl/model/TaskEventImpl.java#L240

**Comment by gmunozfe** [comment]
ArchiveLoggerProvider is not removed, but any service won't be loaded at JPAWorkingMemoryDbLogger.java by default, is it expected behavior?

**Comment by gmunozfe** [comment]
Upgrade community db scripts adding "end_date" columns (jbpm-7.56-to-7.57.sql) should remove these columns in the same way (jbpm-7.59-to-7.60.sql), wdyt?

**Review by gmunozfe — CHANGES_REQUESTED** [review]
Looks good, I've just detected a missing removal column (end_date) for AuditTaskImpl at oracle-springboot-jbpm-schema.sql

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-springboot-jbpm-schema.sql [hunk 1]
```diff
@@ -33,7 +33,7 @@ create sequence VAR_INST_LOG_ID_SEQ start with 1 increment by  1;
 create sequence WORKITEMINFO_ID_SEQ start with 1 increment by  50;
 create table Attachment (id number(19,0) not null, accessType number(10,0), attachedAt timestamp, attachmentContentId number(19,0) not null, contentType varchar2(255 char), name varchar2(255 char), attachment_size number(10,0), attachedBy_id varchar2(255 char), TaskData_Attachments_Id number(19,0), primary key (id));
 create table AuditTaskImpl (id number(19,0) not null, activationTime timestamp, actualOwner varchar2(255 char), createdBy varchar2(255 char), createdOn timestamp, deploymentId varchar2(255 char), description varchar2(255 char), dueDate timestamp, lastModificationDate timestamp, name varchar2(255 char), parentId number(19,0) not null, priority number(10,0) not null, processId varchar2(255 char), processInstanceId number(19,0) not null, processSessionId number(19,0) not null, status varchar2(255 char), taskId number(19,0), workItemId number(19,0), end_date timestamp, primary key (id));
```
[the remark this answers is not in the document]
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-springboot-jbpm-schema.sql:35]: you are right

**Review by gmunozfe — APPROVED** [review]
Looks good to me, amazing work @elguardian !

**Pull request merged**